sherpa/astro/ui/utils.py

```diff
@@ -8860,6 +8864,11 @@ def fake_pha(self, id, arf, rmf, exposure, backscal=None, areascal=None,
            calling ``fake_pha``. That background dataset needs to include
            the data itself (not used in this function), the background model,
            and the response.
+        method : callable or None, optional
+           If None, the default, then the data is simulated using the
+           sherpa.utils.poisson_noise routine. If set, it must be a
```

Comment:
```
`sherpa.utils.poisson_noise` routine. If set, it must be a
```
To make Sphinx render a link.

Reply:
In checking whether this already worked (which it doesn't, but sometimes it does) I see that there's an error in the docstring for the preceeding option which I can fix whilst I'm at it:

![sherpa](https://user-images.githubusercontent.com/224638/235197396-fdd6a638-24b0-41aa-b483-4fc0ef216dfc.png)

Reply:
should be done but I don't want to resolve yet.